src/PackageJsonSynchronizer.php

```diff
@@ -150,6 +149,41 @@ private function registerWebpackResources(array $phpPackages)
         file_put_contents($controllersJsonPath, json_encode($newControllersJson, \JSON_PRETTY_PRINT | \JSON_UNESCAPED_SLASHES)."\n");
     }
 
+    public function registerPeerDependencies(array $phpPackages)
+    {
+        $peerDependencies = [];
+
+        foreach ($phpPackages as $phpPackage) {
+            if (!$assetsDir = $this->resolveAssetsDir($phpPackage)) {
+                continue;
+            }
+
+            $packageJsonPath = $this->rootDir.'/vendor/'.$phpPackage.$assetsDir.'/package.json';
+            $packageJson = (new JsonFile($packageJsonPath))->read();
```

Comment:
These 2 lines could be moved into a private method - they're repeated elsewhere

Reply:
I'm going to keep them as is, at least for now. The private method would need to return two variables, that's not worth the dedup IMHO.

Reply:
Now part of #767